hugo/layouts/papers/single.html

```diff
@@ -362,7 +362,7 @@ <h5 class="modal-title">Correct Metadata for <span id="paperIdSpan"></span></h5>
         <div class="modal-footer d-flex align-items-center">
           <div class="form-check mb-0">
             <input type="checkbox" class="form-check-input" id="pdfCorrectionCheck">
-            <label class="form-check-label" for="pdfCorrectionCheck">ALL author names, the title, and the abstract match the PDF</label>
+            <label class="form-check-label" for="pdfCorrectionCheck">ALL author names, the title, and the abstract match the PDF. If paper metadata matches the PDF, but the grouping of papers into author pages is incorrect, please file an <a href="https://github.com/acl-org/acl-anthology/issues/new?assignees=anthology-assist&labels=correction%2Cmetadata&projects=&template=02-name-correction.yml&title=Author+Page%3A+%7Breplace+with+author+name%7D">author page correction</a> instead.</label>
```

Comment:
I wonder if "grouping of papers into author pages is incorrect" is too opaque. 

Maybe "..., but you want to fix which author page(s) this paper appears on"?

Reply:
I struggled with how to phrase this. What about: "but an author of this paper has published under multiple names and they need to be linked"?

Reply:
But that’s just one possible case, it could be that the author wants to disambiguate themself from others with the same name.

Reply:
"but the linking of papers to unique authors is incorrect"?

Reply:
"but the paper should be linked to a different author page"?

Reply:
OK